store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/operation/ChangeGraphId.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.federatedstore.operation;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.apache.commons.lang3.exception.CloneFailedException;
+
+import uk.gov.gchq.gaffer.commonutil.Required;
+import uk.gov.gchq.gaffer.operation.io.Output;
+import uk.gov.gchq.gaffer.operation.serialisation.TypeReferenceImpl;
+import uk.gov.gchq.koryphe.Since;
+import uk.gov.gchq.koryphe.Summary;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreConstants.KEY_OPERATION_OPTIONS_GRAPH_IDS;
+
+
+@JsonPropertyOrder(value = {"class", "graphId", "newGraphId"}, alphabetic = true)
+@Since("1.10.6")
+@Summary("Changes the protection used for accessing graphs")
+@JsonInclude(Include.NON_DEFAULT)
+public class ChangeGraphId implements Output<Boolean> {
+    @Required
+    private String graphId;
+    private String newGraphId;
+    private Map<String, String> options = new HashMap<>();
+
+    public ChangeGraphId() {
+        addOption(KEY_OPERATION_OPTIONS_GRAPH_IDS, "");
```

Comment:
Is this done elsewhere? Seems like there should be a removeOption in the Operation interface?

Reply:
It replaced the need to stick `""` as default `getCleanStrings(config.get(KEY_OPERATION_OPTIONS_GRAPH_IDS));`

but happy to change it.

Reply:
I don't think there is a remove option in the interface.

Reply:
Could you use getOptions().remove()?

Reply:
Yes :D I forgot I had direct access to the options object. I was working through the Operations interface.

Reply:
Null and "" provide different behaviours.
FederatedStore#getStreams this was a change by @p013570 
This is done is 6 other classes.